app/enterprise/1.3-x/brain-immunity/install-configure.md

```diff
@@ -94,8 +94,14 @@ Then you can bring up collector with:
 SQLALCHEMY_DATABASE_URI=postgres://{POSTGRES-USER}:{POSTGRES-PASSWORD}@{POSTGRES HOST}:{POSTGRES POST}/collector CELERY_BROKER_URL={REDIS URI} docker-compose -f docker-compose.yml up -d
 ```
 
-#### Step 2. Set up the Collector Plugin
-Enable the Collector Plugin using the Admin API:
+#### Step 2. Set up the Collector Plugin and Kong GUI
+Enable Immunity/Brain GUI changes on Kong Manager by editing the kong.env file to have:
```

Comment:
(Not sure if this matters or whether these are equivalent, just going by the PR title.)

```suggestion
Enable Immunity/Brain GUI changes on Kong Manager by editing the kong.conf file to have:
```

Reply:
Oh it is kong.conf!!  Not to be confused with kong.env